### PR TITLE
{lib,tools}[dummy/] Hadoop v2.6.0-cdh5.12.0, snappy v1.1.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2471.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dda05124ffe3eae848a5939b30dd6b00 for a full test report.

File: easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+patches = ['Hadoop-TeraSort-on-local-filesystem.patch']
+
+builddependencies = [
+    ('Maven', '3.5.0'),
+    ('protobuf', '2.5.0'),  # *must* be this version
+    ('CMake', '3.9.1'),
+    ('snappy', '1.1.6'),
+]
+
+dependencies = [('Java', '1.7.0_80')]
```
Comment (verdurin) [inline easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb]: Does it *have* to be this old `Java` version too?

Comment (boegel) [inline easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb]: As I understood, yes, Java 8 is only supported in Hadoop 2.7 & later, I think.

I seem to remember trying with `1.8.0_*`, and running into compilation problems...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/63de763027326f06bcfd9cf33ce61cf4 for a full test report.

File: easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.12.0-native.eb]: Fair enough.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**